osu.Game/Overlays/Rankings/Tables/UserBasedTable.cs

```diff
@@ -32,7 +33,7 @@ protected UserBasedTable(int page, IReadOnlyList<UserStatistics> rankings)
         protected override Drawable CreateHeader(int index, TableColumn column)
         {
             var title = column?.Header ?? string.Empty;
```

Comment:
Can do `column?.Header ?? default` here to avoid the implicit conversion of `string.Empty` to a localisable. This goes for a fair few places in this pull.

Reply:
Have applied this on my own.